src/components/Dashboard/index.js

```diff
@@ -35,7 +35,7 @@ class Dashboard extends React.Component {
 }
 
 const mapStateToProps = state => ({
-  isAuthed: Boolean(state.session.session),
+  isAuthed: Boolean(state.session.user),
```

Comment:
FYI: Just another way to make a truthy or falsy here is using `!!state.session.user` . But that one is alright too! As I see, it already was there before :)

Reply:
I like the !!something way, much cleaner.

Reply:
I may use it too often in my own projects and I heard people complaining about it, but I find it the easiest way to transform any JavaScript literal into a boolean.